### DISCUSSION TITLE
Charge limit not working

### DISCUSSION THREAD

**Discussion by sfa5150** [seerge/g-helper#196] [General]
Hello there. I can't get the charge limit to work. I gave Ghelper full admin privileges in scheduler, but nothing. This is a TUF Dash F15 2022 12650H 3070

Also, is there a way to fully shut off keyboard backlight when switching to battery?

**Comment by seerge** [comment]
Hello, based on 0 information that you have provided about charge limit, i can't help you. Also this is discussions section, not issues. App applies limit setting on launch / every time slider is changed / and when laptop goes to sleep. 

You can turn off keyboard backlit with corresponding checkbox in Keyboard - Extra
